test/oss_test.cpp

```diff
@@ -14,16 +14,75 @@
 */
 
 #include "ossSocket.hpp"
+#include "core.hpp"
 
 #include <iostream>
+#include <unistd.h>	
 #include <gtest/gtest.h>
 
+
+unsigned int genRandomPort() {
+    time_t t; 
+    srand((unsigned) time(&t)); 
+    unsigned int ret = 0; 
+    ret = rand() % (OSS_MAX_PORT - OSS_MIN_PORT + 1) + OSS_MIN_PORT; 
+    return ret; 
+}
+
+bool isPortOccupy(unsigned int port) {
+    return false; 
+    /*
```

Comment:
disable first, find a better way to detect ports conflict instead of calling system's shell command